packages/ember-data/lib/system/relationships/state/relationship.js

```diff
@@ -63,16 +63,6 @@ Relationship.prototype = {
     });
   },
 
-  addCanonicalRecords: function(records, idx) {
-    for (var i=0; i<records.length; i++) {
-      if (idx !== undefined) {
-        this.addCanonicalRecord(records[i], i+idx);
-      } else {
-        this.addCanonicalRecord(records[i]);
-      }
-    }
-  },
```

Comment:
This might be useful later and follows the convention of `addRecord` and `addRecords`. Could you leave this code in and remove the `diff` code and I can merge this change?